autoscheduler/frontend/src/redux/reducers/term.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Stores the current term
+ */
+
+// type
+export class TermType {
```

Comment:
I'm not understanding why we need a `TermType` wrapper around a single number. Is `TermType` planned to include multiple fields in the future?

Reply:
See I don't either, but frankly I couldn't figure out how to store just a single number in the Redux store, but I'll try again

Reply:
Oh nvm I got it working. I swear I could've sworn I tried this exact same thing before but I guess not lmao